src/app/auth/components/auth/auth.component.scss

```diff
@@ -0,0 +1,97 @@
+/* @format */
+@import 'variables';
+
+:host {
+  background: $linear-gradient-background;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  padding: 0px 64px;
+
+  @media screen {
+    padding: 20px 0px;
+  }
+}
+
+.auth-container {
+  display: flex;
+  gap: 64px;
+  width: 100%;
+  align-items: center;
+  overflow-y: scroll;
+
+  @include beforeDesktop {
+    justify-content: center;
+  }
+}
+
+.auth {
+  width: 25%;
+  height: 100%;
+
+  @include beforeDesktop {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+  }
+
+  @media screen and (min-width: 900px) {
+    width: 35%;
+    padding-right: 20px;
+  }
+}
+
+.image {
+  position: relative;
+  width: 80%;
+  height: 100%;
+
+  & > img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
```

Comment:
Should we have a more specific maximum/minimum height for this image? I feel like it has potential to look weird at different zoom levels, on big monitors, or close to the mobile breakpoint.

This is more a question for the QA process though.